+eui/ConditionPanel.m

```diff
@@ -0,0 +1,319 @@
+classdef ConditionPanel < handle
+  %CONDITIONPANEL Deals with formatting trial conditions UI table
+  %   Designed to be an element of the EUI.PARAMEDITOR class that manages
+  %   the UI elements associated with all Conditional parameters.
+  % TODO Add sort by column
+  % TODO Add set condition idx
+  
+  properties
+    % Handle to UI Table that represents trial conditions
+    ConditionTable
+    % Minimum UI Panel width allowed.  See also EUI.PARAMEDITOR/ONRESIZE
+    MinWidth = 80
+%     MaxWidth = 140
+%     Margin = 4
+    % Handle to parent UI container
+    UIPanel
+    % Handle to UI container for buttons
+    ButtonPanel
+    % Handles to context menu items
+    ContextMenus
+  end
+  
+  properties (Access = protected)
+    % Handle to EUI.PARAMEDITOR object
+    ParamEditor
+    % UIControl button for adding a new trial condition (row) to the table
+    NewConditionButton
+    % UIControl button for deleting trial conditions (rows) from the table
+    DeleteConditionButton
+    % UIControl button for making conditional parameter (column) global
+    MakeGlobalButton
+    % UIControl button for setting multiple table cells at once
+    SetValuesButton
+    % Indicies of selected table cells  as array [row, column;...] of each
+    % selected cell
+    SelectedCells 
+  end
+  
+  methods
+    function obj = ConditionPanel(f, ParamEditor, varargin)
+      % FIELDPANEL Panel UI for Conditional parameters 
+      %  Input f may be a figure or other UI container object
+      %  ParamEditor is a handle to an eui.ParamEditor object.
+      % 
+      % See also EUI.PARAMEDITOR, EUI.FIELDPANEL
+      obj.ParamEditor = ParamEditor;
+      obj.UIPanel = uix.VBox('Parent', f);
+%       obj.UIPanel.BackgroundColor = 'white';
```

Comment:
can this be removed?

Reply:
You may recall that the editor was originally white, which I think looks nicer.  MC is still all grey by default.  I can't be arsed to change everything to white so the short answer is yes, let's remove all of these BackgroundColor lines.

Reply:
ha hmm well you're right the white background does look nicer...